packages/client/src/stores/ListStore.ts

```diff
@@ -26,6 +26,11 @@ export default class StackStore<T> {
     return this.data.pop()!;
   }
 
+  @action.bound
+  removeIndex(index: number): void {
+    this.data = observable.array([...this.data.slice(0, index), ...this.data.slice(index + 1)]);
+  }
```

Comment:
What is the use case for this function? When would you want to remove a specific error from the list and would you have the index?

Reply:
https://github.com/Nike-Inc/referee/pull/40/files#diff-a513f662e5fcaa6baa9a7ed6807f918bR26

it's for when you hit the x on an error.

Reply:
Makes sense!